src/UbiProtocolHandler.cpp

```diff
@@ -143,7 +143,7 @@ void UbiProtocolHandler::buildHttpPayload(char *payload) {
   sprintf(payload, "{");
 
   for (uint8_t i = 0; i < _current_value;) {
-    char str_value[20];
+    char str_value[25];
```

Comment:
What error does this prevent?

Reply:
The errors described in this PR for the esp32: https://github.com/ubidots/ubidots-esp32/commit/09cd6f6eaba825392d6e29807ebc54770e2d72e4